src/shared/views/ProjectView.js

```diff
@@ -331,6 +332,7 @@ class _ProjectHeader extends React.Component<HeaderProps, HeaderState> {
             projectType: project.projectType,
         });
         switch (project.projectType) {
+        case COMPLETENESS_PROJECT:
```

Comment:
hey,
just a quick one here: in case of COMPLETENESS_PROJECT nothing happens, right?

Reply:
This basically means "do the same thing as the following". See https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/switch for reference. Here, COMPLETENESS_PROJECT and LEGACY_TILES will do the exact same thing.